### ISSUE TITLE
rviz_python_tutorial myviz.py segmentation fault

### ISSUE THREAD

**Issue by awesomebytes** [ros-visualization/visualization_tutorials#28]
I was starting to get myself familiar with librviz from python and trying the example in rviz_python_tutorial just crashes with segmentation fault.

I do:

```
roscd rviz_python_tutorial
```

(To be in the folder where `config.myviz` is, which will be loaded.)

Then:

```
rosrun rviz_python_tutorial myviz.py
```

And I get the output:

```
[ INFO] [1437573701.223369341]: Stereo is NOT SUPPORTED
[ INFO] [1437573701.223440384]: OpenGl version: 3 (GLSL 1.3).
Segmentation fault (core dumped)
```

After a brief splash of the window.

I tried on two machines with ROS hydro and got the same behaviour. With Rviz 1.10.19. It crashes just after [myviz.show()](https://github.com/ros-visualization/visualization_tutorials/blob/indigo-devel/rviz_python_tutorial/myviz.py#L159).

**Comment by awesomebytes** [comment]
I noticed that the line [74](https://github.com/ros-visualization/visualization_tutorials/blob/indigo-devel/rviz_python_tutorial/myviz.py#L74):

```
self.frame.setStatusBar( None )
```

Has something to do with the crash. If I comment out that line, it works!

**Comment by awesomebytes** [comment]
Ok I just found this ticket: https://github.com/ros-visualization/rviz/issues/895 which mainly is what this own ticket is about

**Comment by wjwwood** [comment]
If you have time, please try https://github.com/ros-visualization/rviz/pull/896 which is a proposed solution to ros-visualization/rviz#895.

**Comment by awesomebytes** [comment]
I applied https://github.com/ros-visualization/rviz/pull/896 to the branch hydro-devel and the `myviz.py` is example is still crashing

**Comment by wjwwood** [comment]
@awesomebytes I found the same to be true, but after trying a few different things I have a new fix (still in ros-visualization/rviz#896) can you give that a try?

I'll probably go ahead and merge it since I observed it fixing the issue. Last time I think I had other changes that made me think it had been fixed when it had not.

**Comment by wjwwood** [comment]
I'm going to close this, please comment here if the problem persists after the change I merged (and released).
